### PR TITLE
[WIP] First incomplete but working and valid version of Swagger (fix #27)

### LINKED ISSUE #27: Swagger?
http://swagger.io/specification/

Not sure it's worth it (high implementation / maintenance cost, real usage to be proven), but let's keep it in the radar.

### PR DESCRIPTION
**Pull request by yohanboniface**
Work in progress.

### CODE REVIEW

File: ban/http/routing.py [hunk 1]
```diff
@@ -44,17 +41,7 @@ def attached_reverse(self, name, req, instance):
                                      reverse(name, identifier=instance.id))
 
     def register_route(self, api):
-        api.add_route('/help', self)
```
Comment (davidbgk) [inline ban/http/routing.py]: No redirection for that removed route?

Comment (yohanboniface) [inline ban/http/routing.py]: We are not in production yet ;)

Comment (davidbgk) [inline ban/http/routing.py]: Fair enough :p

**Comment by yohanboniface** [comment]
Superseded by #198

**Pull request closed without merging**